### PR TITLE
2.6.1 release

### PR DESCRIPTION
**Pull request by t1m0thyj**
Signed-off-by: Timothy Johnson <timothy.johnson@broadcom.com>

prepare 2.6.1 release

### CODE REVIEW

**Review by JillieBeanSim — APPROVED** [review]
Thanks @t1m0thyj LGTM

**Review by zFernand0 — APPROVED** [review]

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,6 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:5]: this should be ## `2.6.1`

**Review by JillieBeanSim — CHANGES_REQUESTED** [review]

File: packages/zowe-explorer/CHANGELOG.md (hunk 1 above)
Comment (zFernand0) [reply packages/zowe-explorer/CHANGELOG.md:5]: I believe the automation will replace this headers too, based on what happened in this commit:
- https://github.com/t1m0thyj/vscode-extension-for-zowe/commit/ea1344c083e096e1bde6a288224c16bc4c2910ff

Comment (JillieBeanSim) [reply packages/zowe-explorer/CHANGELOG.md:5]: ok, I didn't realize it changed the CHANGELOG headers too.

**Review by JillieBeanSim — APPROVED** [review]

**Pull request merged**